cli/internal/plugins/git/ifaces_test.go

```diff
@@ -0,0 +1,23 @@
+package git
+
+import (
+	"context"
+	"os/exec"
+)
+
+type versionRunner struct {
```

Comment:
I feel we could call this something like `hardcodedVersionRunner`, and have another implementation for the CLI version. p.e: `cliVersionRunner` Where we call actual command.